### PR TITLE
Add order shipping address and invoice related attributes documentation

### PR DESCRIPTION
**Pull request by cvrac**

### CODE REVIEW

File: source/localizable/smart_cart/_order_object.html.md.erb [hunk 1]
```diff
@@ -94,3 +107,27 @@ Name                                                       | Type    | Descripti
...
+
+Name                           | Type    | Description
+:----------------------------: | :-----: | :----------------------------------------------:
+`company`                      | String  | Company name
+`profession`                   | String  | Company profession
+`vat_number`                   | String  | Company's vat number
+`doy`                          | String  | Company's public economic service
+`address`                      | String  | Billing address
+`vat_exclusion_requested`      | Boolean | Vat exclusion requested under 39A
+`vat_exclusion_representative` | Object  | [39A representative details]((#vat-exclusion-representative) - if applicable
```
Comment (nikostsaganos) [inline source/localizable/smart_cart/_order_object.html.md.erb:125]: Typo:

image

File: source/localizable/smart_cart/_order_object.html.md.erb [hunk 2]
```diff
@@ -94,3 +107,27 @@ Name                                                       | Type    | Descripti
...
+
+This attribute is present only when a user requests an invoice for the order. A user
+may also request vat exclusion for several line items.
+
+
+Name                           | Type    | Description
+:----------------------------: | :-----: | :----------------------------------------------:
+`company`                      | String  | Company name
+`profession`                   | String  | Company profession
+`vat_number`                   | String  | Company's vat number
```
Comment (nikostsaganos) [inline source/localizable/smart_cart/_order_object.html.md.erb:121]: `VAT number`

File: source/localizable/smart_cart/_order_object.html.md.erb [hunk 3]
```diff
@@ -13,13 +13,26 @@ Name         | Type   | Value | Description
...
 
 #### Customer details
 
 Name                              | Type   | Description
 :-------------------------------: | :----: | :------------------------------------------:
 `customer.first_name`             | String |First name
 `customer.last_name`              | String |Last name
+`customer.address`                | Object |Order shipping address
+
+### Customer address object
```
Comment (nikostsaganos) [inline source/localizable/smart_cart/_order_object.html.md.erb:27]: Let's not use H3 here. another H4 would be fine

image

Also, we could name it "Customer address"

File: source/localizable/smart_cart/_order_object.html.md.erb [hunk 4]
```diff
@@ -13,13 +13,26 @@ Name         | Type   | Value | Description
...
+`invoice`        | Boolean | | Indicates invoice request for the order
+`invoice_details` | Object | | [Invoice details](#invoice-details) - if invoice requested
 
 #### Customer details
 
 Name                              | Type   | Description
 :-------------------------------: | :----: | :------------------------------------------:
 `customer.first_name`             | String |First name
 `customer.last_name`              | String |Last name
+`customer.address`                | Object |Order shipping address
```
Comment (nikostsaganos) [inline source/localizable/smart_cart/_order_object.html.md.erb:25]: We could link the description to the object below.

File: source/localizable/smart_cart/_order_object.html.md.erb [hunk 5]
```diff
@@ -94,3 +107,27 @@ Name                                                       | Type    | Descripti
 `line_item_rejection_reason.id`                            | Integer | The rejection reason ID to use
 `line_item_rejection_reason.label`                         | String  | A human readable label
 `line_item_rejection_reason.requires_available_quantity`   | Boolean | Whether or not `available_quantity` is required
+
+#### Invoice details
```
Comment (nikostsaganos) [inline source/localizable/smart_cart/_order_object.html.md.erb:111]: Move right after customer address (also in code).

File: source/localizable/smart_cart/_order_object.html.md.erb [hunk 6]
```diff
@@ -94,3 +107,27 @@ Name                                                       | Type    | Descripti
...
+
+This attribute is present only when a user requests an invoice for the order. A user
```
Comment (nikostsaganos) [inline source/localizable/smart_cart/_order_object.html.md.erb:113]: I think we may remove this: `A user may also request vat exclusion for several line items.`

File: source/localizable/smart_cart/_order_object.html.md.erb [hunk 7]
```diff
@@ -94,3 +107,27 @@ Name                                                       | Type    | Descripti
...
+
+
+Name                           | Type    | Description
+:----------------------------: | :-----: | :----------------------------------------------:
+`company`                      | String  | Company name
+`profession`                   | String  | Company profession
+`vat_number`                   | String  | Company's vat number
+`doy`                          | String  | Company's public economic service
+`address`                      | String  | Billing address
+`vat_exclusion_requested`      | Boolean | Vat exclusion requested under 39A
```
Comment (nikostsaganos) [inline source/localizable/smart_cart/_order_object.html.md.erb:124]: I suggest we write it like this:
`VAT exclusion requested (Υπηρεσία άρθρου 39α)`, με το `Υπηρεσία άρθρου 39α` link στο https://www.aade.gr/epiheiriseis/forologikes-ypiresies/fpa/yperesia-arthroy-39a

(39α is greek-only)

File: source/localizable/smart_cart/_order_object.html.md.erb (hunk 3 above)
Comment (cvrac) [reply source/localizable/smart_cart/_order_object.html.md.erb:27]: This was by mistake :) +1 for the description!

File: resources/responses/skroutz/merchants_ecommerce_orders_show.yml [hunk 8]
```diff
@@ -25,7 +25,9 @@
     X-Cache: MISS
     X-Cached: skroutz miss(0)
   :body: '{"order":{"code":"191128-1234567","state":"open",
-"customer":{"first_name":"John","last_name": "Doe"},
+  "customer":{"first_name":"John","last_name": "Doe","address":{"street_name":"Πανεπιστημίου","street_number":"4","zip":"12345","city":"Αθήνα","region":"Αττικής"}},
+"invoice": true,
+"invoice_details": {"company":"Company name","profession":"Company profession","vat_number":"111119191","doy":"Company doy","address": {"street_name":"Πανεπιστημίου","street_number":"4","zip":"12345","city":"Αθήνα","region":"Αττικής"},"vat_exclusion_requested":true,"vat_exclusion_representative": {"id_type":"ΑΤ Αστυνομική Ταυτότητα","id_number":"XX00040","otp":"123456"}}},
```
Comment (nikostsaganos) [inline resources/responses/skroutz/merchants_ecommerce_orders_show.yml:30]: id_number should be 5 digits only

File: source/localizable/smart_cart/_order_object.html.md.erb [hunk 9]
```diff
@@ -13,13 +13,48 @@ Name         | Type   | Value | Description
 `dispatch_until` | Date String in format<br> `YYYY-MM-DDTHH:MM:SS+HH:M` | | Order maximum dispatch date
 `accept_options` | Object | | [Accept options](#accept-options) (available for orders with state `"open"`)
 `reject_options` | Object | | [Reject options](#reject-options) (available for orders with state `"open"`)
+`invoice`        | Boolean | | Indicates invoice request for the order
```
Comment (nikostsaganos) [inline source/localizable/smart_cart/_order_object.html.md.erb:16]: Move after "customer".

Also move comments after invoice_details

File: source/localizable/smart_cart/_order_object.html.md.erb [hunk 10]
```diff
@@ -13,13 +13,48 @@ Name         | Type   | Value | Description
...
+`invoice_details` | Object | | [Invoice details (optional)](#invoice-details) - if invoice requested
```
Comment (nikostsaganos) [inline source/localizable/smart_cart/_order_object.html.md.erb:17]: ✂️ ` - if invoice requested`

File: source/localizable/smart_cart/_order_object.html.md.erb [hunk 11]
```diff
@@ -13,13 +13,48 @@ Name         | Type   | Value | Description
...
+
+This attribute is present only when a user requests an invoice for the order.
+
+Name                           | Type    | Description
+:----------------------------: | :-----: | :----------------------------------------------:
+`company`                      | String  | Company name
+`profession`                   | String  | Company profession
+`vat_number`                   | String  | Company's VAT number
+`doy`                          | String  | Company's public economic service
+`address`                      | String  | Billing address
```
Comment (nikostsaganos) [inline source/localizable/smart_cart/_order_object.html.md.erb:47]: Link to a new "object" table

File: source/localizable/smart_cart/_order_object.html.md.erb [hunk 12]
```diff
@@ -13,13 +13,48 @@ Name         | Type   | Value | Description
...
+`vat_exclusion_requested`      | Boolean | VAT exclusion requested ([Υπηρεσία άρθρου 39α](https://www.aade.gr/epiheiriseis/forologikes-ypiresies/fpa/yperesia-arthroy-39a))
+`vat_exclusion_representative` | Object  | [39A representative details (optional)](#vat-exclusion-representative) - if applicable
```
Comment (nikostsaganos) [inline source/localizable/smart_cart/_order_object.html.md.erb:49]: ✂️ ` - if applicable`

File: source/localizable/smart_cart/_order_object.html.md.erb [hunk 13]
```diff
@@ -13,13 +13,48 @@ Name         | Type   | Value | Description
...
+
+#### Vat exclusion representative
+
+Name         | Type   | Description
+:----------: | :----: | :----------------------------------------------:
+`id_type`    | String | Provided id document type
+`id_number`  | String | Provided id document number
```
Comment (nikostsaganos) [inline source/localizable/smart_cart/_order_object.html.md.erb:56]: Let's explain it's last 5 digits

**Pull request merged**